packages/openapi/src/decorators.ts

```diff
@@ -29,3 +29,22 @@ export function $extension(program: Program, entity: Type, extensionName: string
 export function getExtensions(entity: Type): ReadonlyMap<string, any> {
   return openApiExtensions.get(entity) ?? new Map<string, any>();
 }
+
+// The @default decorator can be applied to a model. When that model is used
+// as the return type of an operation, this return type will be the default response.
+const defaultResponses = new Set<Type>();
+export function $default(program: Program, entity: Type) {
+  if (entity.kind !== "Model") {
+    reportDiagnostic(program, {
```

Comment:
Depending on if this get merged first or the decorator test/validation https://github.com/microsoft/cadl/pull/236 there is a `validateDecoratorTarget` util function that does that and emit the diagnostic for you. 
"decorator-wrong-type" has also been removed from library